package.json

```diff
@@ -46,6 +46,7 @@
   "dependencies": {
     "history": "^3.0.0",
     "lodash.assign": "^4.2.0",
+    "lodash.isequal": "^4.4.0",
```

Comment:
I brought it in to maintain es5 support, although it's weird that `lodash` just deprecated it, considering its usage in the browser: https://twitter.com/jdalton/status/779739251143348224

Maybe a small polyfill is the answer here.